### PR TITLE
feat: add edit link for receipt info

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[電子收據] 個人專區 / 贊助紀錄: 新增修改贊助資料](https://app.asana.com/0/1203699264568808/1207629974038944/f)

# Notice

# Dependency

### CODE REVIEW

File: src/components/member-page/donation/table/row-detail.js [hunk 1]
```diff
@@ -188,10 +202,14 @@ export const TableRowDetail = memo(({ record, periodicHistory, isLoading }) => {
     is_anonymous: isAnonymous,
     pay_method: payMethod,
     order_number: orderNumber,
+    type,
   } = record
   const receiptAddress = `${record.address_state || ''}${record.address_city ||
     ''}${record.address_detail || ''}`
 
+  const { releaseBranch } = useContext(CoreContext)
+  const editReceiptUrl = `${origins.forClientSideRendering[releaseBranch].support}/contribute/${contributeType[type]}/${orderNumber}/utm_source=supportsuccess&utm_medium=account`
```
Comment (Aylie-Chou) [inline src/components/member-page/donation/table/row-detail.js:211]: `utm` should be query string ( `/` -> `?`)

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
as comment~~

File: src/components/member-page/donation/table/row-detail.js (hunk 1 above)
Comment (liruchen32) [reply src/components/member-page/donation/table/row-detail.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**